locations/jclouds/src/test/java/org/apache/brooklyn/policy/jclouds/os/CreateUserPolicyLiveTest.java

```diff
@@ -73,6 +73,7 @@ public void runTestCreatesUser(MachineProvisioningLocation<SshMachineLocation> l
             app.createAndManageChild(EntitySpec.create(TestEntity.class)
                     .policy(PolicySpec.create(CreateUserPolicy.class)
                             .configure(CreateUserPolicy.GRANT_SUDO, true)
+                            .configure(CreateUserPolicy.RESET_LOGIN_USER, true)
```

Comment:
Why use `RESET_LOGIN_USER=true` here? If anyone were to run `testIntegrationCreatesUser()`, it would reset their user password! We should either delete `testIntegrationCreatesUser()`, or remove this (to rely on the default of false).